src/main/java/pl/sda/library/domain/port/AuthorRepository.java

```diff
@@ -17,9 +17,6 @@
 
     void deleteAuthor(long id);
 
-    Author editAuthor(Author author);
+    Optional<Author> editAuthor(Author author);
```

Comment:
jestesmy juz w kontekscie autora - w koncu to authorRepository
wystarczy samo edit

Reply:
poprawione